src/analysis/connectivity/score_inputs.sql

```diff
@@ -0,0 +1,2636 @@
+----------------------------------------
+-- INPUTS
+-- location: neighborhood
+----------------------------------------
+DROP TABLE IF EXISTS generated.neighborhood_score_inputs;
+
+CREATE TABLE generated.neighborhood_score_inputs (
+    id SERIAL PRIMARY KEY,
+    category TEXT,
+    score_name TEXT,
+    score NUMERIC(16,4),
+    notes TEXT,
+    human_explanation TEXT,
+    use_pop BOOLEAN,
+    use_emp BOOLEAN,
+    use_k12 BOOLEAN,
+    use_tech BOOLEAN,
+    use_univ BOOLEAN,
+    use_doctor BOOLEAN,
+    use_dentist BOOLEAN,
+    use_hospital BOOLEAN,
+    use_pharmacy BOOLEAN,
+    use_retail BOOLEAN,
+    use_grocery BOOLEAN,
+    use_social_svcs BOOLEAN,
+    use_parks BOOLEAN,
+    use_trails BOOLEAN,
+    use_comm_ctrs BOOLEAN,
+    use_transit BOOLEAN
+);
+
+
+-------------------------------------
```

Comment:
That makes sense, and I saw some examples of that while perusing the PR. Just wanted to provide an alternative, as coming back to this massive multiple-thousands of lines file for further development could be frustrating in the future.